core/src/main/java/org/jivesoftware/sparkimpl/profile/AvatarPanel.java

```diff
@@ -291,7 +293,49 @@ public void setParentDialog(Dialog dialog) {
         this.dlg = dialog;
     }
 
+    /*
+     * Crop the given image file to the target dimensions 
+     * without changing aspect ratio.
+     * Save the resultant file into disk and return a reference
+     * to it.  
+     */
+    private File cropImageFile(File selectedFile){
+
+        int targetImageWidth = 100;
+        int targetImageHeight = 100;
```

Comment:
The old code was using 128x128px. Some 128x128 images actually works, some not(may be because of file size constraints.). 100x100 works for most images. But using 64x64 will be safe and standard.